app/src/main/java/com/shagalalab/constitution/data/dao/ArticleDao.kt

```diff
@@ -0,0 +1,12 @@
+package com.shagalalab.constitution.data.dao
+
+import androidx.room.Dao
+import androidx.room.Query
+import com.shagalalab.constitution.data.models.ArticleModel
+
+@Dao
+interface ArticleDao {
+
+    @Query("SELECT * FROM article")
```

Comment:
Sorry, my bad, the table names should be plural, i.e. `articles` here, and in other places, try that one.